lib/apis/places.js

```diff
@@ -18,6 +18,50 @@
 var utils = require('../internal/convert');
 var v = require('../internal/validate');
 
+/**
+ * A Find Place request takes a text input, and returns a place.
+ * The text input can be any kind of Places data, for example,
+ * a name, address, or phone number.
+ *
+ * @memberof! GoogleMapsClient
+ * @name GoogleMapsClient.findPlaces
+ * @function
+ * @param {Object} query
+ * @param {string} query.input
+ * @param {string} query.inputtype
+ * @param {string} [query.language]
+ * @param {Array<string>} [query.fields]
+ * @param {ResponseCallback} callback Callback function for handling the result
+ * @return {RequestHandle}
+ */
+exports.findPlaces = {
+  url: 'https://maps.googleapis.com/maps/api/place/findplacefromtext/json',
+  validator: v.compose([
+    v.object({
+      input: v.string,
+      inputtype: v.oneOf(['textquery', 'phonenumber']),
+      language: v.optional(v.string),
+      fields: v.optional(utils.arrayOf(v.oneOf([
+        'alt_id', 'formatted_address', 'geometry', 'icon', 'id', 'name',
```

Comment:
can we remove alt_id? I dont think this API supports it.

Reply:
Done in e0956a9e591fbbd30e86b8d53cb0ac3cc5248ed2